### DISCUSSION TITLE
Save form on tab click

### DISCUSSION THREAD

**Discussion by juliermefelix** [serenity-is/Serenity#6811] [Help]
Save form on tab click
Dear,
I need that when I click on the "Perfil" tab, the form that is on the "Candidato" tab is saved.
Any suggestions on how to do it
![image](https://github.com/serenity-is/Serenity/assets/7516443/ea991ae0-3fe6-4def-810b-6cba18458ab5)

I tried to put the code in this tab method
![image](https://github.com/serenity-is/Serenity/assets/7516443/fb53d5ae-0daf-4248-a76d-dcbc257a704b)

**Comment by ShifanilaSenthil** [comment]
On each tab, write a method and call the service call method (save response)
